test/main.sh

```diff
@@ -220,7 +223,7 @@ if [ "${1:-"all"}" != "cluster" ]; then
     run_test test_tls_jwt "JWT authentication"
 fi
 
-if [ "${1:-"all"}" != "standalone" ]; then
+if [ "${1:-"all"}" != "standalone" ] && [ "${INCUS_BACKEND}" != "truenas" ]; then
```

Comment:
Our test selection logic is quite messy right now, but instead of this, I think I'd prefer a check be added earlier in the "Available storage backends" logic to instantly fail if the testsuite is called with INCUS_BACKEND=truenas and `${1} != "standalone`, effectively explicitly requiring that the testsuite be run in standalone mode when testing truenas.